src/features/world/scenes/PlazaScene.ts

```diff
@@ -484,101 +499,146 @@ export class PlazaScene extends BaseScene {
     });
   }
 
-  async create() {
-    super.create();
-
-    const IS_EASTER = hasFeatureAccess(this.gameState, "EASTER");
-
-    if (IS_EASTER) {
-      this.anims.create({
-        key: `glint_anim`,
-        frames: this.anims.generateFrameNumbers("glint", {
-          start: 0,
-          end: 6,
-        }),
-        repeat: -1,
-        frameRate: 6,
-        repeatDelay: 800,
+  setUpFactionBanners() {
+    // Add banners
+    this.bumpkinsBanner = this.add
+      .image(40, 17, "bumpkins_banner")
+      .setDepth(1000000);
+    this.goblinsBanner = this.add
+      .image(100, 17, "goblins_banner")
+      .setDepth(1000000);
+    this.nightshadesBanner = this.add.image(90, 60, "nightshades_banner");
+    this.nightshadesBannerTop = this.add
+      .image(90, 56, "nightshades_banner_top")
+      .setDepth(1000000);
+    this.sunfloriansBanner = this.add
+      .image(35, 60, "sunflorians_banner")
+      .setDepth(1000000);
```

Comment:
```suggestion
    this.bumpkinsBanner = this.add
      .image(40, 17, "bumpkins_banner")
      .setDepth(17);
    this.goblinsBanner = this.add
      .image(100, 17, "goblins_banner")
      .setDepth(17);
    this.nightshadesBanner = this.add.image(90, 60, "nightshades_banner");
    this.nightshadesBannerTop = this.add
      .image(90, 56, "nightshades_banner_top")
      .setDepth(56);
    this.sunfloriansBanner = this.add
      .image(35, 60, "sunflorians_banner")
      .setDepth(60);
```

Setting depth = y may help banners display more accurately

Before:
![1](https://github.com/sunflower-land/sunflower-land/assets/41215134/7404fdcd-1d3b-4a01-b3af-3d4eadd79ab7)


After:
![2](https://github.com/sunflower-land/sunflower-land/assets/41215134/61ffbd47-6117-446e-93f5-bf1a9e440c74)